src/lib/product_reviews/visualizer/index.js

```diff
@@ -62,6 +63,44 @@ class Visualizer extends Component {
       const { average, groupedReviews } = productReviews;
       const numReviews = productReviews.count;
 
+      const schemaReviews = productReviews.reviews
+         .slice(0,10)
+         .map(r => {
+            return {
+               "@type": "Review",
+               "reviewRating": {
+                  "@type": "Rating",
+                  "ratingValue": `${r.rating}`,
```

Comment:
Any reason we're using string interpolation on all these? Seems like they could be
```suggestion
                  "ratingValue": r.rating,
```

Reply:
Originally I was worried that they needed to be strings, but since we do stringify later on I don't think we need to interpolate :+1: